go-controller/pkg/node/node.go

```diff
@@ -31,6 +31,11 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+const (
+	// Annotations used by multiple external gateways feature
+	externalGatewayPodIPsAnnotations = "k8s.ovn.org/external-gw-podIPs"
+)
```

Comment:
Per below let's have one location for this annotation definition, not two.

Reply:
yep fixed this, moved it into util package.